### PR TITLE
add name

### PR DESCRIPTION
**Pull request by tcclevenger**

### CODE REVIEW

File: 9.1/paper.tex [hunk 1]
```diff
@@ -783,7 +790,7 @@ \section{Acknowledgments}
   Vishal Boddu,
   Benjamin Brands,
   Pi-Yueh Chuang,
-  T. Conrad Clevenger,
+  Thomas C. Clevenger,
```
Comment (tjhei) [inline 9.1/paper.tex]: I think you need to remove yourself here.

File: 9.1/paper.tex [hunk 2]
```diff
@@ -48,7 +48,8 @@
     Matthias Maier,
     Jean-Paul Pelteret,
     Bruno Turcksin,
-    David Wells
+    David Wells,
+    Thomas C. Clevenger
```
Comment (tjhei) [inline 9.1/paper.tex]: alphabetical order?!

**Comment by tcclevenger** [comment]
Also, not sure if I should change the author numbers to be in consecutive order?

**Comment by tcclevenger** [comment]
I added two numbers to Timo, and updated the numbers in ascending order.

**Comment by bangerth** [comment]
I guess we'll need to let @tjhei weigh in whether these really are his soon-to-be coordinates.

**Comment by tjhei** [comment]
👍

**Pull request merged**